src/System/Contexts/SystemContext.tsx

```diff
@@ -20,7 +18,8 @@ export interface SystemContextState {
 }
 
 export interface SystemContextProps extends SystemContextState {
-  unleashClient?: Unleash | UnleashClient
+  isEnabled?: (flag: string) => boolean
+  getVariant?: (flag: string) => any
```

Comment:
Can we move this under a `featureFlags` namespace? 

```tsx
featureFlags: {
  isEnabled: ...
  getVariant: ... 
}
```

This is then accessed via context values that are self-describing:

```
match.context.featureFlags.isEnabled("foo")
match.context.featureFlags.getVariant("bar")
```

Future things can be added here unconfusingly if there's a namespace:
```tsx
match.context.featureFlags.getAll // etc
```

Reply:
definitely, I had this thought too